docs/connectors/morningstar/security-compare.md

```diff
@@ -48,6 +49,26 @@ const connector = new HighchartsConnectors.Morningstar.SecurityCompareConnector(
 });
 ```
 
+### viewIds
+For some converters (Currently MarketCap), it is important to explicitly
+set the `viewIds` property to `'test'` in order to access the relevant data.
```

Comment:
I think we should rephrase it a little. The `test` (and `HSsnapshot`) were created for us. Other customers may use different view names. I think the best would be to send users to Morningstar documentation: https://developer.morningstar.com/direct-web-services/documentation/direct-web-services/security-details/investment-details#get-views (with sentence like "Use the appropriate [view](https://developer.morningstar.com/direct-web-services/documentation/direct-web-services/security-details/investment-details#get-views) that supports MarketCap capabilities. For more details, contact [Morningstar Direct Web Services](https://www.morningstar.com/business/brands/data-analytics/products/direct-web-services#contactus).").

The same about Compare docs